zabbix/README.md

```diff
@@ -57,18 +57,43 @@ modifications.
 In order to configure the monitoring of the Rudder nodes, you can provide Zabbix with parameters (macros) associated to its
 hosts. The "Monitoring parameter" generic method achieves this.
 
```

Comment:
This file actually needs to be converted to asciidoc to be integrated in the doc (see https://raw.githubusercontent.com/Normation/rudder-plugins/master/centreon/README.adoc as reference)